lib/disability_compensation/loggers/monitor.rb

```diff
@@ -62,6 +62,46 @@ def track_saved_claim_save_success(claim, user_account_uuid)
         )
       end
 
+      # Logs toxic exposure data changes during Form 526 submission
+      #
+      # Compares InProgressForm toxic exposure data with submitted claim data
+      # to detect when toxic exposure data has been changed or removed by the frontend.
+      # Logs which specific keys were removed to validate purge logic.
+      #
+      # @param in_progress_form [InProgressForm] User's saved form data
+      # @param submitted_claim [SavedClaim::DisabilityCompensation::Form526AllClaim] The submitted claim
+      # @param submission [Form526Submission] The submission record
+      # @param user_uuid [String] User's UUID
+      # @return [void]
+      def track_toxic_exposure_changes(in_progress_form:, submitted_claim:, submission:, user_uuid:)
+        in_progress_form_data = parse_form_data(in_progress_form.form_data)
+        submitted_data = parse_form_data(submitted_claim.form)
+        return unless in_progress_form_data && submitted_data
+
+        # NOTE: Both InProgressForm.form_data and SavedClaim.form store only the inner
+        # content of the form526 submission (without the 'form526' wrapper).
+        # The wrapper only exists in the original HTTP request body.
+        # Keys use camelCase format (toxicExposure, gulfWar1990, etc.)
+        in_progress_toxic_exposure = in_progress_form_data['toxicExposure']
+        submitted_toxic_exposure = submitted_data['toxicExposure']
```

Comment:
@freeheeling there's no PII in this metadata potentially right? Although I suppose there could be at some point?

Reply:
@NB28VT If I understand correctly, only removed keys or a `completely_removed` boolean value will be logged.